<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Système de Gestion Logistique</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabClients">
       <attribute name="title">
        <string>Clients</string>
       </attribute>
       <layout class="QVBoxLayout" name="vlc">
        <item>
         <layout class="QFormLayout" name="formClient">
          <item row="0" column="0">
           <widget class="QLabel" name="lblNom">
            <property name="text">
             <string>Nom</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leNom"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblPrenom">
            <property name="text">
             <string>Prénom</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lePrenom"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblTel">
            <property name="text">
             <string>Téléphone</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leTel"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblEmail">
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leEmail"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lblAdr">
            <property name="text">
             <string>Adresse</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="leAdr"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="lblStatut">
            <property name="text">
             <string>Statut</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="cbStatut">
            <item>
             <property name="text">
              <string>ACTIVE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>INACTIVE</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="filterRowClient">
          <item>
           <widget class="QLineEdit" name="leSearchClient">
            <property name="placeholderText">
             <string>Rechercher (nom, prénom, email)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leEmailFilter">
            <property name="placeholderText">
             <string>Filtrer par email</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbStatutFilter">
            <item>
             <property name="text">
              <string>Tous</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ACTIVE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>INACTIVE</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbClientSort">
            <item>
             <property name="text">
              <string>Trier par</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Nom</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Prénom</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Créé</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAdvSearchClient">
            <property name="text">
             <string>Recherche Avancée</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="btnRowClient">
          <item>
           <widget class="QPushButton" name="btnAddClient">
            <property name="text">
             <string>Ajouter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUpdClient">
            <property name="text">
             <string>Modifier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelClient">
            <property name="text">
             <string>Supprimer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRefClient">
            <property name="text">
             <string>Rafraîchir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClientStats">
            <property name="text">
             <string>Statistiques</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnExportExcel">
            <property name="text">
             <string>Exporter Excel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUpdateCategory">
            <property name="text">
             <string>Modifier Catégorie</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tblClients">
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
          <property name="columnCount">
           <number>8</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nom</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Prénom</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Téléphone</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Email</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Adresse</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Statut</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Créé le</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="clientChartContainer" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabOrders">
       <attribute name="title">
        <string>Commandes</string>
       </attribute>
       <layout class="QVBoxLayout" name="vlo">
        <item>
         <layout class="QFormLayout" name="formOrder">
          <item row="0" column="0">
           <widget class="QLabel" name="lblClient">
            <property name="text">
             <string>Client</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbClient"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblEtat">
            <property name="text">
             <string>État</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cbEtat">
            <item>
             <property name="text">
              <string>EN_ATTENTE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EN_COURS</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LIVREE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ANNULEE</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblMontant">
            <property name="text">
             <string>Montant</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="dsMontant">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblAdrLiv">
            <property name="text">
             <string>Adresse Livraison</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leAdrLiv"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lblPriority">
            <property name="text">
             <string>Priorité</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="cbPriority">
            <item>
             <property name="text">
              <string>LOW</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>NORMAL</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>HIGH</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>URGENT</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="filterRowOrder">
          <item>
           <widget class="QLineEdit" name="leSearchOrder">
            <property name="placeholderText">
             <string>Rechercher (état, adresse)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbClientFilter">
            <item>
             <property name="text">
              <string>Tous les clients</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbEtatFilter">
            <item>
             <property name="text">
              <string>Tous</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EN_ATTENTE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EN_COURS</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LIVREE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ANNULEE</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="amountFilterLayout">
            <item>
             <widget class="QLabel" name="lblMinAmount">
              <property name="text">
               <string>Min:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dsMinAmount">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblMaxAmount">
              <property name="text">
               <string>Max:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dsMaxAmount">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QComboBox" name="cbOrderSort">
            <item>
             <property name="text">
              <string>Trier par</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Date</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Montant</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>État</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAdvSearchOrder">
            <property name="text">
             <string>Recherche Avancée</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="btnRowOrder">
          <item>
           <widget class="QPushButton" name="btnAddOrd">
            <property name="text">
             <string>Ajouter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUpdOrd">
            <property name="text">
             <string>Modifier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelOrd">
            <property name="text">
             <string>Supprimer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRefOrd">
            <property name="text">
             <string>Rafraîchir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPdfOrd">
            <property name="text">
             <string>Exporter PDF</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnOrderStats">
            <property name="text">
             <string>Statistiques</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUpdatePriority">
            <property name="text">
             <string>Modifier Priorité</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tblOrders">
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
          <property name="columnCount">
           <number>7</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Client</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>État</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Montant</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Adresse Livraison</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Priorité</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="orderChartContainer" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
